src/TreeSitter/Node.hs

```diff
@@ -20,70 +26,107 @@ data Node = Node
   , nodeNamedChildCount :: !Int32
   , nodeChildCount :: !Int32
   }
-  deriving (Show, Eq, Generic, CStorable)
+  deriving (Show, Eq, Generic)
 
 data TSPoint = TSPoint { pointRow :: !Int32, pointColumn :: !Int32 }
-  deriving (Show, Eq, Generic, CStorable)
+  deriving (Show, Eq, Generic)
 
+data TSNode = TSNode !(Ptr ()) !Word32 !Word32
```

Comment:
1. `TSNode` dropped one element from its array field.
2. The array fields are unsigned, so `Word32` is more appropriate.